src/test/java/com/wikia/webdriver/testcases/specialpagestests/FilePageTests.java

```diff
@@ -133,19 +133,21 @@ public void filePage005_deleteFromHistory() {
     // Load the file page again, should have the same name
     filePage.open(video.getWikiFileName()).verifyEmbeddedVideoIsPresent();
 
-    // Go to the history tab and verify there are at least two videos
-    filePage.selectHistoryTab();
-    filePage.verifyVersionCountAtLeast(2);
+    //Removed following lines until SUS-428 is fixed
+    //// Go to the history tab and verify there are at least two videos
+   //filePage.selectHistoryTab();
+    //filePage.verifyVersionCountAtLeast(2);
 
-    // Delete the second version
-    DeletePageObject deletePage = filePage.deleteVersion(2);
-    deletePage.submitDeletion();
+    //// Delete the second version
+    //DeletePageObject deletePage = filePage.deleteVersion(2);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)